test/index.js

```diff
@@ -3,26 +3,12 @@
 var test = require('tape');
```

Comment:
No reason not to make the test loader itself an ES module, and to use eg async functions even in due course too.